src/js/apps/patients/patient/flow/flow_app.js

```diff
@@ -71,33 +73,44 @@ export default SubRouterApp.extend({
       'destroy': this.onActionDestroy,
     });
 
-    this.listenTo(this.flow, 'change:_owner', this.onFlowChangeOwner);
+    this.listenTo(this.flow, {
+      'change:_owner': this.onFlowChangeOwner,
+      'message': this.onFlowMessage,
```

Comment:
@nmajor25 so all messages that are for the resource get retriggered on the resource